llm_on_ray/inference/serve.py

```diff
@@ -148,6 +150,12 @@ def main(argv=None):
         action="store_true",
         help="Whether to keep serve terminal.",
     )
+    parser.add_argument(
+        "--ray_max_concurrent_queries", required=True, type=int, help="The batch size in Ray."
```

Comment:
need better help. the batch size here causes more confusions. just say "Max concurrent requests the serving system can process."